lib/lwip/lwipopts.h

```diff
@@ -27,6 +27,9 @@
 #define LWIP_NETCONN 0
 #define LWIP_SOCKET 0
 
+#ifdef _WIN32
+#define LWIP_NORAND 1
```

Comment:
was this for some specific purpose? might be nice to leave a comment behind as to 'why'?